benchmarks/high_volume_writes_test.go

```diff
@@ -13,35 +14,41 @@ import (
 )
 
 var _ = Describe("Scenario: High Volume Writes", func() {
-	var (
-		scenarioCfg config.HighVolumeWrites
-		beforeOnce  sync.Once
 
-		totalSamples int
-		mu           sync.Mutex // Guard total samples taken before tear down in AfterEach
+	var (
+		scenarioCfg        config.HighVolumeWrites
+		configurationCount int
+		mu                 sync.Mutex // Guard
+		totalSamples       int
 	)
 
-	BeforeEach(func() {
+	BeforeSuite(func() {
 		scenarioCfg = benchCfg.Scenarios.HighVolumeWrites
 		if !scenarioCfg.Enabled {
 			Skip("High Volumes Writes Benchmark not enabled!")
-
 			return
 		}
+	})
```

Comment:
Are you sure this does not skip all test cases if you disable only `HighVolumeWrites`?

Reply:
This skips all the senarios for HighVolumeWrites but should continue to run all other tests

Reply:
Works as expected